### ISSUE TITLE
Bonfire menu dropdown not working on android chrome browser

### ISSUE THREAD

**Issue by colonelo** [ci-bonfire/Bonfire#1271]
I testing application bonfire. It seems to me excellent for development, 
When I tried to run the application on the phone on google chrome  I have a problem that menu navigation but does not do away and stands  the same page. Menu is not performed. Page remains in the same places where is initialy run.

Example:
If the page defualt  in the url:
Bonfire-master_2/public/index.php/admin/reports/activities
and go to menu sifranti 
public/index.php/admin/reports/sifranti
there is nothing to happend, html not navigated  (url in chrome android) not change.

On the PC windows system this working correctly with same version of application

![image](https://user-images.githubusercontent.com/24300396/26890920-88e97b20-4bb3-11e7-9bf0-f7e60100df3e.png)

Thanks.

Matej

**Comment by GintaWahyudi** [comment]
> I testing application bonfire. It seems to me excellent for development,
> When I tried to run the application on the phone on google chrome I have a problem that menu navigation but does not do away and stands the same page. Menu is not performed. Page remains in the same places where is initialy run.
> 
> Example:
> If the page defualt in the url:
> Bonfire-master_2/public/index.php/admin/reports/activities
> and go to menu sifranti
> public/index.php/admin/reports/sifranti
> there is nothing to happend, html not navigated (url in chrome android) not change.
> 
> On the PC windows system this working correctly with same version of application
> 
> ![image](https://user-images.githubusercontent.com/24300396/26890920-88e97b20-4bb3-11e7-9bf0-f7e60100df3e.png)
> 
> Thanks.
> 
> Matej

I also experienced the same thing, hopefully Bonfire will get updates about bootstrap from version 2 to version 3 soon.

**Comment by colonelo** [comment]
Hi.
Sorry, this did not turn out to be a Bonfire issue. It is a known issue with Bootstrap. Applied this fix and it is resolved:
https://github.com/ci-bonfire/Bonfire/issues/1271
or

I have fixed the ones below in my application. whic is written from
 yeshansachithak commented on 16 Jul 2018 •


Ci-Bonfire still does have the issue. Due to the OLD  version of bootstrap 2.3.2. However, I have fixed the issue with a  simple CSS. public/themes/admin/header.php 
Assets::add_css(array( 'bootstrap.css', 'bootstrap-responsive.css', 'fixmobile.css',//My custom CSS file ));
 
public/themes/admin/css/fixmobile.css /*Fix mobile issue - 16072018*/ .dropdown-backdrop { display: none !important; } 
Hope helps someone. Thanks!
--





Now it works
